projects/1.16/assets/earth-reworked/earthworks/lang/zh_cn.json

```
@@ -0,0 +1,292 @@
+  "block.earthworks.rammed_earth": "夯土",
+  "block.earthworks.rammed_earth_slab": "夯土台阶",
+  "block.earthworks.rammed_earth_stairs": "夯土楼梯",
+  "block.earthworks.rammed_earth_wall": "夯土墙",
+  "block.earthworks.reed": "芦苇",
+  "block.earthworks.reed_slab": "芦苇台阶",
+  "block.earthworks.reed_stairs": "芦苇楼梯",
+  "block.earthworks.reed_wall": "芦苇墙",
+  "block.earthworks.sand_gabion": "沙子铁笼",
+  "block.earthworks.sand_gabion_slab": "沙子铁笼台阶",
+  "block.earthworks.sand_gabion_stairs": "沙子铁笼楼梯",
+  "block.earthworks.sand_gabion_wall": "沙子铁笼墙",
+  "block.earthworks.spruce_timber": "云杉木材",
+  "block.earthworks.spruce_timber_slab": "云杉木材台阶",
+  "block.earthworks.spruce_timber_stairs": "云杉木材楼梯",
+  "block.earthworks.spruce_timber_wall": "云杉木材墙",
+  "block.earthworks.spruce_wood_shakes": "云杉木粗瓦",
+  "block.earthworks.spruce_wood_shakes_slab": "云杉木粗瓦台阶",
+  "block.earthworks.spruce_wood_shakes_stairs": "云杉木粗瓦楼梯",
+  "block.earthworks.spruce_wood_shakes_wall": "云杉木粗瓦墙",
+  "block.earthworks.thatch": "茅草",
+  "block.earthworks.thatch_slab": "茅草台阶",
+  "block.earthworks.thatch_stairs": "茅草楼梯",
+  "block.earthworks.thatch_wall": "茅草墙",
+  "block.earthworks.vertical_acacia_fence": "竖纹金合欢栅栏",
+  "block.earthworks.vertical_acacia_planks": "竖纹金合欢木板",
+  "block.earthworks.vertical_acacia_slab": "竖纹金合欢台阶",
+  "block.earthworks.vertical_acacia_stairs": "竖纹金合欢楼梯",
+  "block.earthworks.vertical_birch_fence": "竖纹白桦栅栏",
+  "block.earthworks.vertical_birch_planks": "竖纹白桦木板",
+  "block.earthworks.vertical_birch_slab": "竖纹白桦台阶",
+  "block.earthworks.vertical_birch_stairs": "竖纹白桦楼梯",
+  "block.earthworks.vertical_dark_oak_fence": "竖纹深色橡栅栏",
+  "block.earthworks.vertical_dark_oak_planks": "竖纹深色橡木板",
+  "block.earthworks.vertical_dark_oak_slab": "竖纹深色橡台阶",
+  "block.earthworks.vertical_dark_oak_stairs": "竖纹深色橡楼梯",
+  "block.earthworks.vertical_jungle_fence": "竖纹丛林栅栏",
+  "block.earthworks.vertical_jungle_planks": "竖纹丛林木板",
+  "block.earthworks.vertical_jungle_slab": "竖纹丛林台阶",
+  "block.earthworks.vertical_jungle_stairs": "竖纹丛林楼梯",
+  "block.earthworks.vertical_oak_fence": "竖纹橡栅栏",
+  "block.earthworks.vertical_oak_planks": "竖纹橡木板",
+  "block.earthworks.vertical_oak_slab": "竖纹橡台阶",
+  "block.earthworks.vertical_oak_stairs": "竖纹橡楼梯",
+  "block.earthworks.vertical_spruce_fence": "竖纹云杉栅栏",
+  "block.earthworks.vertical_spruce_planks": "竖纹云杉木板",
+  "block.earthworks.vertical_spruce_slab": "竖纹云杉台阶",
+  "block.earthworks.vertical_spruce_stairs": "竖纹云杉楼梯",
+  "block.earthworks.wattle_and_daub": "抹灰篱笆",
+  "block.earthworks.wattle_and_daub_slab": "抹灰篱笆台阶",
+  "block.earthworks.wattle_and_daub_stairs": "抹灰篱笆楼梯",
+  "block.earthworks.wattle_and_daub_wall": "抹灰篱笆墙",
+  "block.earthworks.wicker": "柳条",
+  "block.earthworks.wicker_slab": "柳条台阶",
+  "block.earthworks.wicker_stairs": "柳条楼梯",
+  "block.earthworks.wicker_wall": "柳条墙",
+  "item.earthworks.acheulean": "阿舍利石器工具",
+  "item.earthworks.adobe_brick": "粘土砖块",
+  "item.earthworks.adz": "锛子",
+  "item.earthworks.chalk_dust": "白垩岩粉末",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "item.earthworks.chalk_dust": "粉笔灰",
```

---

游戏内为建材，用于制造泥墙方块